test/api/services/MtdIdLookupServiceSpec.scala

```diff
@@ -27,7 +27,7 @@ class MtdIdLookupServiceSpec extends ServiceSpec {
     lazy val target = new MtdIdLookupService(mockMtdIdLookupConnector)
   }
 
-  val nino: String        = "AA123456A"
+  val nino: String = "AA123456A"
   val invalidNino: String = "INVALID_NINO"
 
```

Comment:
Please add type annotations for val expected and connectorResponse in this file.